pom.xml

```diff
@@ -86,7 +86,7 @@
         <picocli.version>4.6.2</picocli.version>
         <pme.version>4.6</pme.version>
 
-        <pnc.version>2.4.1</pnc.version>
+        <pnc.version>2.5.0-SNAPSHOT</pnc.version>
```

Comment:
Note this changes the version of PNC ... I can revert that if needed but for the instrumentation to function we need this.

Reply:
Yup it's fine!